lib/streamlit/__init__.py

```diff
@@ -190,9 +191,41 @@ def _reset(main_dg, sidebar_dg):
 warning = _with_dg(_DeltaGenerator.warning)  # noqa: E221
 
 # Config
-set_option = _config.set_option
+
 get_option = _config.get_option
 
+
+def set_option(key, value):
+    """Set config option.
+
+    Currently, only two config options can be set within the script itself:
+        * client.caching 
+        * client.displayEnabled 
+
+    Calling with any other options will raise StreamlitAPIException.
+
+    Run `streamlit config show` in the terminal to see all available options.
+
+    Parameters
+    ----------
+    key : str
+        The config option key of the form "section.optionName". To see all
+        available options, run `streamlit config show` on a terminal.
+
+    value
+        The new value to assign to this config option.
+
+    """
+    opt = _config._config_options[key]
+    if opt.scriptable:
+        _config.set_option(key, value)
+        return
+
+    raise StreamlitAPIException(
+        '{key} cannot be set on the fly. Set as command line option, e.g. streamlit run script.py --{key}, or in config.toml instead.'.format(key=key)
```

Comment:
Is the single quote needed here? We mostly use double quotes for strings elsewhere. I guess black allows that because of the interpolation, right?

Reply:
Good catch, that's just my habit!  I'll change to double.